src/act.comm.cpp

```diff
@@ -1128,10 +1170,12 @@ ACMD(do_broadcast)
           else
             strlcat(radio_string, "^n", sizeof(radio_string));
 
-          store_message_to_history(d, COMM_CHANNEL_RADIO, act(radio_string, FALSE, ch, 0, d->character, TO_VICT));
+          store_message_to_history(d, COMM_CHANNEL_RADIO, act(radio_string, FALSE, ch, 0, d->character, 
+            d->character->persona && d->character->persona->decker && d->character->persona->decker->deck ? TO_DECK : TO_VICT));
```

Comment:
You're using a much more involved check here (persona, decker, and deck) vs above (just persona). Which do you want to standardize on?

Reply:
persona